content/english/blog/post-20210726-1.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Deployment of an OSCAR cluster in the EGI Federated Cloud"
+date: 2021-07-25T13:00:00+01:00
+# post image
+image: "images/blog/post-20210726-1/portada_blog_oscar_synergy.png"
+# post type (regular/featured)
+type: "featured"
+# meta description
+description: "This is meta description"
+# post draft
+draft: false
+---
+
+Here we present a step by step guide to help you deploy an OSCAR cluster using EGI credentials in the [EOSC-Synergy VO](https://infra.eosc-synergy.eu/vos/#synergy). We are using the [IM Dashboard](https://appsgrycap.i3m.upv.es:31443/im-dashboard/login), a tool developed by the [GRyCAP](https://www.grycap.upv.es/) research group at the [Universitat Politècnica de València](https://www.upv.es/) to facilitate the deployment of infrastructures in a lot of cloud providers. Alternatively, you can follow our YouTube video, at the end of the post.
```

Comment:
Consider adding a link to EGI - https://www.egi.eu

Reply:
Added.